subsys/shell/backends/shell_telnet.c

```diff
@@ -275,13 +316,11 @@ static void telnet_accept(struct net_context *client,
 
 	sh_telnet->client_ctx = client;
 
-	/* Disable echo - if command handling is enabled we reply that we don't
+
```

Comment:
Nit: this (second) newline is not needed here.

Reply:
You're right